features/old/services/multiservice.feature

```diff
@@ -43,7 +43,7 @@ Feature: Multiservice feature
     And I fill in "System name" with "SystemName@123"
     And I press "Create Product"
     Then I should see "Invalid."
-    Then I should see the flash message "Couldn't create Product. Check your Plan limits"
+    Then I should see the flash message "System name Invalid."
```

Comment:
After removing the `Invalid.` translation, this will need to be changed to "System name invalid"

Reply:
```suggestion
    And I should see the flash message "System name invalid"
```

Reply:
And the line `:45` should also change: `Then I should see "Invalid"`